src/assets/Keycloak/ocis-with-keycloak.md

```diff
@@ -0,0 +1,161 @@
+---
+title: keycloak Server
```

Comment:
if this blog is focused on keycloak with ocis, then the title should say something about ocis